Sources/ArcGISToolkit/Components/Offline/OfflineMapAreasView.swift

```diff
@@ -89,6 +89,8 @@ public struct OfflineMapAreasView: View {
                     PreplannedListItemView(model: preplannedMapModel) { newMap in
                         selectedMap = newMap
                         dismiss()
+                    } onMapDeletion: {
```

Comment:
```suggestion
                    } onDelete: {
```
SwiftUI uses the view modifier name `onDelete` for several of their views.